modular_skyrat/modules/mapping/code/fluff.dm

```diff
@@ -54,9 +54,34 @@
 		7 = 330,
 		8 = 380
 	)
-/obj/item/tape/ruins/ghostship	//An early 'AI' that gained self-awareness, praising the Machine God. Yes, this whole map is a Hardspace Shipbreaker reference.
+
+/obj/item/tape/ruins/ghostship
+	icon_state = "tape_yellow"
+	desc = "The tape is hastily labelled with \"WARNING\"... might be worth listening to this one."
```

Comment:
```suggestion
	desc = "The tape is hastily labelled with \"WARNING\"... Might be worth listening to this one."
```

Reply:
intentional, ellipses =/= periods, so its not a new sentance